docs/_themes/clusterhq/layout_docs.html

```diff
@@ -0,0 +1,197 @@
+        <div class="row">
+            <!-- Contents menu -->
+            <div class="col-sm-3 page contents-menu">
+                <div class="visible-xs-block mob-contents">
+                    Contents
+                    <div class="pull-right plus">
+                        <a href="javascript:$('.contents-menu>ul').slideToggle();">+</a>
+                    </div>
+                </div>
+                <!-- Menu area start -->
+                {{ toctree(collapse=false, maxdepth=4) }}
+                <!-- Menu area end -->
+            </div>
+            
+            <!-- Main contents area -->
+            <div class="col-sm-8 col-sm-offset-1 page white-bg">
+                {%- if pagename != "search" %}
+                <form class="search" action="{{ pathto('search') }}" method="get" _lpchecked="1">
+                    <input type="text" name="q" placeholder="Type here to search docs..." id="search">
+                    <input type="hidden" name="check_keywords" value="yes">
+                    <input type="hidden" name="area" value="default">
+                </form>
+                {%- endif %}
+                <!-- DOCS PAGE DATA HERE -->
+                {%- block document %}
+                        <script>
+                            var release_re = /^[0-9]+\.[0-9]+\.[0-9]+$/
+                            if (!'{{ release }}'.match(release_re)) {
+                                document.write(
+                        '<div class="admonition note warning"> \
+                            <p class="last"> \
+                                You are reading an <strong>in-development version</strong> \
+                                of the documentation. \
+                                Some of the functionality may not work as expected.</p> \
+                        </div>');
+                            }
+                        </script>
+                        {% block body %} {% endblock %}
+                {%- endblock %}
+                <!-- END OF DOCS PAGE DATA -->
+                <div class="row">
+                    <div class="col-xs-12 col-sm-6">
+                        {%- if prev %}
+                            <a href="{{ prev.link|e }}" title="{{ prev.title|striptags|e }}" class="button rel">Previous</a>
+                        {%- endif %}
+                    </div>
+                    <div class="col-xs-12 col-sm-6">
+                        {%- if next %}
+                            <a href="{{ next.link|e }}" title="{{ next.title|striptags|e }}" class="button rel pull-right">Next</a>
+                        {%- endif %}
+                    </div>
+                </div>
+                
+                <!-- Disqus -->
+                {% if theme_disqus_shortname %}
+                    <div id="disqus_thread"></div>
+                    <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
```

Comment:
There's an awful lot of JavaScript on this page.  Why single out disqus for this support and ignore the rest?  Is the page usable at all without JavaScript?  For example, you can't switch between the platform tabs without JavaScript (clicking silently does nothing).